examples/streaming/redis/src/main/scala/org/apache/gearpump/streaming/example/redis/RedisSourceSink.scala

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.gearpump.streaming.example.redis
+
+import io.gearpump.Message
+import io.gearpump.cluster.UserConfig
+import io.gearpump.cluster.embedded.EmbeddedCluster
+import io.gearpump.cluster.main.ArgumentsParser
+import org.apache.gearpump.streaming.redis.{RedisSink, RedisSource, RedisMessage}
+import io.gearpump.streaming.sink.DataSinkProcessor
+import io.gearpump.streaming.source.DataSourceProcessor
+import io.gearpump.streaming.task.{Task, TaskContext}
+import io.gearpump.streaming.{Processor, StreamApplication}
+import io.gearpump.util.Graph._
+import io.gearpump.util.{AkkaApp, Graph}
+
+class RedisSourceSinkUpperProcessor(taskContext: TaskContext, conf: UserConfig)
+  extends Task(taskContext, conf) {
+
+  import taskContext.output
+
+  override def onNext(message: Message): Unit = {
+    val msg = message.msg.asInstanceOf[Option[String]]
```

Comment:
this would be better as
```
Option(message.msg) match {
  case Some(msg) =>
    val upper = msg.asInstanceOf[String].toUpperCase
    LOG.info(s"to Upper $upper")
    outer(new Message(upper, message.timestamp))
  case None =>
}
```